app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -121,13 +120,10 @@ def link
         end
         @area_name = params[:area_name]
         @content_id = params[:content_id]
-        @attachments = Attachment.all.collect { |f| [f.name, download_attachment_path(:id => f.id, :name => f.urlname)] }
-        if params[:link_urls_for] == "newsletter"
-          @url_prefix = current_server
-        end
-        if multi_language?
-          @url_prefix = "#{Language.current.code}/"
+        @attachments = Attachment.all.collect do |f|
```

Comment:
Style/BlockDelimiters: Prefer `{...}` over `do...end` for functional blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)